write.cc

```diff
@@ -5,43 +5,52 @@
 #include <string>
 #include "messages.pb.h"
 
-using namespace std;
+//using namespace std;
+
 
 int main(int argc, char *argv[]){
     // Verify the version of the library
     GOOGLE_PROTOBUF_VERIFY_VERSION;
 
     int x, y, vel_x, vel_y;
-    FILE *fout;
-    string output;
+    //FILE *fout;
+    ::std::string output;
```

Comment:
http://stackoverflow.com/a/33102382

Check this out... here there was no particular need but during protobuf usage, there might be some nested std namespace, so I preferred it.